src/components/Sidebar.js

```diff
@@ -1,25 +1,37 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
-import Drawer from 'material-ui/Drawer'
-import logo from '../images/logo.svg'
-import LogoutLink from './LogoutLink'
+import React from 'react';
+import { withStyles } from 'material-ui/styles';
 
-/*
-  Renders the IAR application side bar.
+import LogoutLink from './LogoutLink';
+import Divider from 'material-ui/Divider';
+import Toolbar from 'material-ui/Toolbar';
+import Typography from 'material-ui/Typography';
+import List from 'material-ui/List';
+import SidebarNavLink from './SidebarNavLink';
+
+const styles = theme => ({
+  drawerHeader: theme.mixins.toolbar,
+  nested: { paddingLeft: theme.spacing.unit * 4 },
+});
+
+/**
+ * The content of the IAR application side bar.
  */
-const Sidebar = () => (
-  <Drawer variant="permanent">
-    <img src={logo} className="App-logo" alt="logo" />
-    <h3>Assets</h3>
-    <ul>
-      <li><Link to="/assets/dept">My department</Link></li>
-      <li><Link to="/assets/all">All</Link></li>
-      <hr/>
-      <li><Link to="/help">Help</Link></li>
-      <hr/>
-      <li><LogoutLink>Sign out</LogoutLink></li>
-    </ul>
-  </Drawer>
+const Sidebar = ({ classes, history, logout }) => (
+  <div>
```

Comment:
The sidebar height is currently constrained to the height of the Page component. For example when there are no/or little assets in the table.

We could force the Page component to 100% height, however it's probably easier to fix the Sidebar instead if we every need to use the dynamic Page size for styling.

Adding a class to the root of the Sidebar component div with `height: 100vh` should do the trick.